servers/rendering/shader_language.cpp

```diff
@@ -9078,7 +9092,7 @@ Error ShaderLanguage::_parse_shader(const HashMap<StringName, FunctionInfo> &p_f
 										}
 									}
```

Comment:
IMO it makes more sense for both `hint_enum` and `hint_flags` to allow trailing commas, like how they're allowed on function parameter definitions. Unlike `hint_range`, the number of arguments is arbitrary, and for long strings, it can make sense to place them on separate lines, where a trailing comma is appropriate.

Reply:
I'm all for trailing commas... but they aren't in gdshader syntax currently, and `hint_enum` is already committed, so it'd probably be better to have a separate proposal/PR for introducing trailing commas in general, no?